content/md/supplier-onboarder/handle-category.md

```diff
@@ -0,0 +1,102 @@
+---
+id: handle-category
+themes: manage-products
+title: Handle **categories**
+popular: false
+---
+
+# Definition of a category
+In Akeneo Onboarder, you can use **a category to classify products**. A category is always part of a category tree (or *classification tree*).
+
+As an Onboarder user, you can only have **one category tree** in Akeneo with an **unlimited number of levels** (categories, subcategories, sub-subcategories).
+
+![image of the category trees](../img/Settings_CategoryTree.png)
+
+## Differences between families and categories
+In this section, we define two principles of the Akeneo Onboarder: *Categories* and *Families*.
+### Categories
+
+- Trees and categories are a way to classify your products  
+- A product can be in several or no categories
```

Comment:
active form -> you can classify a product in one or several categories. It's is possible for a product to not be in a category, in that case, it goes to the 'Unclassified' Category by default.